configure

```diff
@@ -432,6 +432,11 @@ parser.add_option('--without-ssl',
     dest='without_ssl',
     help='build without SSL')
 
+parser.add_option('--without-node-options',
```

Comment:
You want this implemented for Windows? If so, you'll have to give me a hint as to how. Also, I'm not sure it has to be added. Lots of other `configure` options aren't implemented in vcbuild.bat, I'm not sure this one is so useful it has to exist for Windows.